### PR TITLE
Update bootc-fetch-apply-updates.timer with comments for RandomizedDe…

### PR DESCRIPTION
**Pull request by mrguitar**
…laySec=

RandomizedDelaySec= is the best kept secret of timers. Given that such few people know about it and it's very useful for large deployments, I think it makes sense to add it as a comment.

Thoughts about providing this out of the box as a comment? I'm not proposing we turn it on by default.

### CODE REVIEW

**Comment by jeckersb** [comment]
/ok-to-test

:+1: from me, reminds me of messing with cfengine splaytime many moons ago!

**Review by cgwalters — APPROVED** [review]
I'm in favor of turning it on by default actually...there's not really a strong downside right?

**Comment by mrguitar** [comment]
@jeckersb exactly! I always describe this as "splay"!

@cgwalters, I can't think of a downside. Maybe the comment should change to this if we enable it: "When deploying a large number of systems, it may be beneficial to increase this value to help with load on the registry."

I *think* 2h is a sane default, but I just put that as an example. Should it change?

**Comment by mrguitar** [comment]
ok, I tried to make that change, but since I suck at git it's stuck on DCO. :) This is what PM help looks like. lol

**Comment by cgwalters** [comment]
See https://docs.pi-hole.net/guides/github/how-to-signoff/

TL;DR do `git commit --amend --signoff`

**Comment by cgwalters** [comment]
I force pushed an update to this:

- squashed the commits
- Tweaked the commit message now that it's not a comment
